Source/Controllers/SPAppController.m

```diff
@@ -984,6 +987,41 @@ - (void)handleEventWithURL:(NSURL*)url
         [result writeToFile:resultFileName atomically:YES encoding:NSUTF8StringEncoding error:nil];
         return;
     }
+    
+    if ([command isEqualToString:@"LaunchFavorite"]) {
+        NSString *targetBookmarkName = nil;
+        NSURLComponents *components = [NSURLComponents componentsWithURL:url resolvingAgainstBaseURL:NO];
+        for (NSURLQueryItem *queryItem in components.queryItems) {
+            if ([queryItem.name isEqualToString:@"name"]) {
+                targetBookmarkName = queryItem.value;
+                break;
+            }
+        }
+        
+        if (targetBookmarkName && [targetBookmarkName length]) {
+            SPTreeNode *targetFavoriteNode = nil;
+            SPTreeNode *favoritesTree = [SPFavoritesController sharedFavoritesController].favoritesTree;
+            for (SPTreeNode *favoriteNode in [favoritesTree allChildLeafs]) {
+                if ([favoriteNode.dictionaryRepresentation[SPFavoriteNameKey] isEqualToString:targetBookmarkName]) {
+                    targetFavoriteNode = favoriteNode;
+                    break;
+                }
+            }
+            
+            if (targetFavoriteNode) {
+                SPWindowController *windowController = [self.tabManager newWindowForWindow];
+                SPFavoritesOutlineView *favoritesOutlineView = windowController.databaseDocument.connectionController.favoritesOutlineView;
+                [favoritesOutlineView selectRowIndexes:[NSIndexSet indexSetWithIndex:[favoritesOutlineView rowForItem:targetFavoriteNode]] byExtendingSelection:NO];
+                [windowController.databaseDocument.connectionController initiateConnection:windowController.databaseDocument.connectionController];
+                return;
+            }
+        }
+        
+        NSBeep();
+        [NSAlert createWarningAlertWithTitle:NSLocalizedString(@"LaunchFavorite URL Scheme Error", @"LaunchFavorite URL Scheme Error") message: [NSString stringWithFormat:@"%@ %@ “%@”", NSLocalizedString(@"The variable in the ?name= query parameter could not be matched with any of your favorites.", @"The variable in the ?name= query parameter could not be matched with any of your favorites."), NSLocalizedString(@"Variable: ", @"Variable: "), targetBookmarkName] callback:nil];
```

Comment:
Looks great! If you can add these strings to the English lang file (only need to add to the English lang file, the rest are managed automatically) this should then be ready to go!

Reply:
Ok - I've omitted variable since it was translated already